packages/css/src/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import type { CSSInterpolation } from '@emotion/serialize'
```

Comment:
q: is this file needed? it seems that it shouldn't be there (probably no `packages/*/src/index.d.ts` should exist, nor it should be committed). It has been introduced (accidentally?) in https://github.com/emotion-js/emotion/pull/2655/files#diff-2554203f690453f2a59501ba1a911173d872db1d2a47743f5befd0e60819436b but it was not a part of the original `@emotion/css` migration: https://github.com/emotion-js/emotion/pull/2558/files

Reply:
Yeah those were definitely added unintentionally. I think preconstruct is creating superfluous `.d.ts` files on Windows... will look into this.

Reply:
There is a chance that this is something that I've fixed here: https://github.com/preconstruct/preconstruct/pull/366 and that was released in `@preconstruct/cli@2.0.3` while Emotion is still using v1.

I've prepared a PR with the upgrade ([here](https://github.com/emotion-js/emotion/pull/2734)) - could you recheck if this fixes this problem on your machine?